### DISCUSSION TITLE
Is there any backend with demo data ?

### DISCUSSION THREAD

**Discussion by zeromask1337** [realworld-apps/realworld#717] [Q&A] [answered]
I found original backend quite laggy. For example, today it hung or worked intermittently. Are there any ready-made backends with demo data, e.g. users, posts, likes?

**Comment by geromegrignon** [comment, marked as answer]
Hello @zeromask1337,

We are working to deploy a new server on the existing domain.
As a temporary more stable solution , you can use https://realworld-temp-api.herokuapp.com/api

There is no demo data as the database is reset on Sundays.
To seed it with demo data is part of my backlog.

**Comment by cirosantilli** [comment]
Mine has it BTW: https://github.com/cirosantilli/node-express-sequelize-nextjs-realworld-example-app/blob/7def72741eb4123deed9b5e956967d5ca6ba2f6e/bin/generate-demo-data.js but not through API, directly via database.

Related: https://github.com/gothinkster/realworld/issues/77

**Comment by geromegrignon** [comment]
I added more fake data with comments: https://demo.realworld.io/#/
